pages/common/az-login.md

```diff
@@ -0,0 +1,29 @@
+# az login
+
+> Login to Azure.
+> Part of `az`.
+> More information: <https://docs.microsoft.com/cli/azure/reference-index#az_login>.
+
+- Login interactively:
+
+`az login`
+
+- Login with a username and password:
+
+`az login --username {{alias@somedomain.com}} --password {{secret}}`
```

Comment:
Is there any way to input the password interactively? Generally, we try to avoid examples where your credentials may land in your history

Reply:
Not sure if there is one :( 
but
 `az login` is interactive though

Reply:
Maybe we should put a warning in the description like
`> Inputting a password directly into a command will save it in the shell's history, accesible to anyone.`

Reply:
I do not recommend including an example that shows that the password can be directly specified on the command line at all. If `az login` is interactive, then we should have that as an example instead.

Reply:
Lets nuke this example then :)

Reply:
What about the others with `--password` can you just leave them out and enter the password interactively?

Reply:
No I am not sure if we can. I tried an example which did not work. 
Cant comment on if there is a way to do it :(